network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_teaser_block.html

```diff
@@ -13,9 +13,11 @@
         {% if card.article %}
           <div class="tw-px-4 tw-w-full medium:tw-w-1/3 tw-mb-5">
             <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
-              <a href="{% pageurl card.article %}">
-                <span class="tw-h5-heading tw-mb-2">{{ card.article.title }}</span>
-              </a>
+              <span class="tw-h5-heading tw-mb-2 tw-block">
```

Comment:
Can you add some context why we are using `span` with `tw-block` instead of just using a `div`?

Reply:
Normally we would use spans to style text but in this instance yea a div makes more sense.  However i've made a further change realizing we don't need the span or the wrapping div for this link either so i've updated this further and removed unnecessary elements.